docs/src/content/docs/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Welcome to XTDB!
+---
+
+import Fiddle from '@components/fiddle.astro';
+import Txs from '@components/fiddle/txs.astro';
+import Query from '@components/fiddle/query.astro';
+import { Aside } from '@astrojs/starlight/components';
+
+XTDB is an immutable SQL database that reduces the time and cost of building and maintaining safe systems of record. You can read more about our mission [here](/intro/why-xtdb).
+
+In XTDB there's no need for handcrafted "audit tables", or bespoke versioning and filtering logic - just write regular looking SQL and a granular history of all changes will be preserved (and be accessible for whenever you need it!):
+
+<Fiddle magicContext="my-context">
+  <Txs systemTime="2024-01-01"
+       txs="INSERT INTO product (_id, name, price) VALUES
+            (1, 'An Electric Bicycle', 400)" />
+  <Txs systemTime="2024-01-05"
+       txs="UPDATE product SET price = 405 WHERE _id = 1" />
+  <Txs systemTime="2024-01-10"
+       txs="UPDATE product SET price = 350 WHERE _id = 1" />
+  <Query q={`SELECT *, _valid_from
+FROM product
+  FOR VALID_TIME ALL -- i.e. "show me all versions"
+  FOR SYSTEM_TIME AS OF DATE '2024-01-31' -- "...as observed at month end"`} />
+</Fiddle>
+
+Unlike regular SQL databases, XTDB retains all history by default and helps businesses to easily and accurately report: _**“Here is the complete history of my data, as I understood it previously, and as I understand it currently.”**_
+
+XTDB builds upon the 'bitemporal' capabilities defined within [SQL:2011](https://en.wikipedia.org/wiki/SQL:2011) but makes those capabilities both ubiquitous and automatic, without imposing any unnecessary complications on the application schema.
+
+<Aside title="Looking for 1.x (stable release)?">
+    <p>This is the documentation site for XTDB version 2, which is currently in early access.</p>
+    <p>To find out more about early access and how 2.x relates to 1.x, see <a href="https://xtdb.com/v2" target="_blank">here</a>.</p>
+    <p>For documentation on the stable, production-ready 1.x releases of XTDB, see <a href="https://v1-docs.xtdb.com" target="_blank">here</a>.</p>
+</Aside>
+
+## Learn XTDB in 10 minutes
+
+To discover the novel features of XTDB, take a look at the interactive [SQL Quickstart](/quickstart/sql-overview).
+
+Or alternatively, get XTDB running locally via Docker first before working through the Quickstart, by following the steps [here](/intro/installation-via-docker).
```

Comment:
I would make the whole `following the steps here` the link, more eye catching :)